app/controlplane/api/controlplane/v1/group.proto

```diff
@@ -0,0 +1,157 @@
+//
+// Copyright 2025 The Chainloop Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package controlplane.v1;
+
+import "buf/validate/validate.proto";
+import "controlplane/v1/pagination.proto";
+import "controlplane/v1/response_messages.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/chainloop-dev/chainloop/app/controlplane/api/controlplane/v1;v1";
+
+// GroupService provides operations for managing groups within the system
+service GroupService {
+  // Create creates a new group with the specified name and description
+  rpc Create(GroupServiceCreateRequest) returns (GroupServiceCreateResponse) {}
+  // Get retrieves a specific group by its ID
+  rpc Get(GroupServiceGetRequest) returns (GroupServiceGetResponse) {}
+  // List retrieves a paginated list of groups, with optional filtering
+  rpc List(GroupServiceListRequest) returns (GroupServiceListResponse) {}
+  // Update modifies an existing group's attributes
+  rpc Update(GroupServiceUpdateRequest) returns (GroupServiceUpdateResponse) {}
+  // Delete removes a group from the system
+  rpc Delete(GroupServiceDeleteRequest) returns (GroupServiceDeleteResponse) {}
+  // ListMembers retrieves the members of a specific group
+  rpc ListMembers(GroupServiceListMembersRequest) returns (GroupServiceListMembersResponse) {}
+}
+
+// GroupServiceCreateRequest contains the information needed to create a new group
+message GroupServiceCreateRequest {
+  // Name of the group to create
+  string name = 1 [(buf.validate.field).string.min_len = 1];
+  // Description providing additional information about the group
+  optional string description = 2 [(buf.validate.field).ignore_empty = true];
```

Comment:
technically you do not need to make optional a field during creation since you do not care if it's empty or nil, from the creation POV it's the same, for the update it's another story though.